aspnetcore/blazor/file-uploads.md

```diff
@@ -64,4 +78,21 @@ The following example demonstrates multiple image file upload in a component:
 }
 ```
 
-To read data from a user-selected file, call `OpenReadStream` on the file and read from the returned stream. In a Blazor WebAssembly app, the data is streamed directly into the .NET code within the browser. In a Blazor Server app, the file data is streamed into .NET code on the server as the file is read from the stream. 
+`IBrowserFile` returns metadata [exposed by the browser](https://developer.mozilla.org/docs/Web/API/File#Instance_properties) as properties. This metadata can be useful to preliminary validation. For example, see the [`FileUpload.razor` and `FilePreview.razor` sample components](https://github.com/dotnet/AspNetCore.Docs/tree/master/aspnetcore/blazor/file-uploads/samples/).
+
+## File streams
+
+In a Blazor WebAssembly app, the data is streamed directly into the .NET code within the browser.
+
+In a Blazor Server app, the file data is streamed over the SignalR connection into .NET code on the server as the file is read from the stream. In the following table, `Forms.RemoteBrowserFileStreamOptions` allows configuring file upload characteristics for Blazor Server.
+
+| Blazor Server Property | Description | Default |
+| ---------------------- | ----------- | ------- |
+| `MaxBufferSize` | Maximum internal buffer size for unread data sent over a SignalR circuit. | 20,480 KB (20 KB) |
```

Comment:
This looks nice. I thought this option type didn't warrant a whole lot of explanation because it's kinda difficult to figure out what tweaking some of these means to a user's code. It's hard to tell why one would change some of these unless users came up with very specific issues. The other bit is that adding the defaults would mean we would have to remember to update these if the code is updated. How do you feel about dropping that?

Reply:
Ok ... I'll drop it. They'll eventually have a cross-link here to the API doc, and I'll get that in there after GA.